playbooks/README.md

```diff
@@ -107,11 +107,6 @@ The playbook expects you have ```oc```, rpm package ```origin-clients```,
 installed and a config file in ```$HOME/.kube/config```.  Also make sure you
 indentified with the cluster using ```oc login```.
 
-Before installing KubeVirt on an existing OpenShift cluster, ensure that SELinux is disabled on all hosts:
```

Comment:
Maybe worth rephrasing - or adding to some "Known issues": "In case you are experienceing permission or SELinux issues you can disable selinux by running …"